Dockerfile

```diff
@@ -1,8 +1,11 @@
-FROM ruby:2.6.5-stretch
+FROM ruby:3.2.2-slim
 
 WORKDIR /home/gusto
 
 ADD . /home/gusto/
 
+RUN apt-get update && apt-get install -y --no-install-recommends \
```

Comment:
this is required for CI to pass, I'll ask prod-infra whether these changes are fine or not